include/pms-utils/vdb/vdb.hpp

```diff
@@ -0,0 +1,121 @@
+#pragma once
+
+#include "pms-utils/atom/atom.hpp"
+#include "pms-utils/depend/depend.hpp"
+#include "pms-utils/ebuild/ebuild.hpp"
+
+#include <cstdint>
+#include <expected>
+#include <filesystem>
+#include <format>
+#include <ostream>
+#include <variant>
+
+namespace [[gnu::visibility("default")]] pms_utils {
+namespace vdb {
+
+struct Obj {
+    std::filesystem::path path;
+    std::string md5;
+    std::uint64_t size;
+};
+
+struct Dir {
+    std::filesystem::path path;
+};
+
+struct Sym {
+    std::filesystem::path src;
+    std::filesystem::path dest;
+    std::uint64_t size;
+};
+
+using Content = std::variant<Obj, Dir, Sym>;
```

Comment:
no using decls in public headers!!!

make this a struct that inherits from it instead